build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
It looks like the integration test is failing because of the issue that @richardmcclellan [encountered a few weeks ago](https://github.com/aws-amplify/amplify-android/pull/981#issuecomment-739519528) when he tried to update the gradle plugin.

The error message appears to be the same:

```
| 1612941665257 | FAILURE: Build failed with an exception.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
| 1612941665257 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
| 1612941665257 | * What went wrong:                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
| 1612941665257 | Could not determine the dependencies of task ':core:l8DexDesugarLibDebugAndroidTest'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
| 1612941665257 | > Cannot query the value of this property because it has no value available.       
```

Reply:
Ah yes, thanks! I forgot the exact versioning and was hoping enough time had passed :-).

I'll do another cut of the PR that omits the AGP bump for now.